source/normalize-arguments.ts

```diff
@@ -60,43 +61,40 @@ export const preNormalizeArguments = (options: Options, defaults?: Options): Nor
 	}
 
 	delete options.timeout;
-
 	const {retry} = options;
+
 	options.retry = {
-		retries: () => 0,
-		methods: new Set(),
-		statusCodes: new Set(),
-		errorCodes: new Set(),
-		maxRetryAfter: undefined
+		...defaultOptions.options.retry as RetryOption,
```

Comment:
This is wrong. It should be `defaults.retry` so we get to the same point as before.